.prettierignore

```diff
@@ -3,3 +3,4 @@ docs
 lib
 pnpm-lock.yaml
 test/.webpack
+*.md
```

Comment:
What did you run into to ignore Markdown for prettier?

Reply:
It was checking CHANGELOG.md file when running test.
That's why I added this line.
Here is a screenshot of it.

<img width="991" alt="image" src="https://github.com/dbartholomae/middy-middleware-class-validator/assets/14869419/e0816c4f-4c75-4ab9-9e68-d3e44c2a2fd0">